<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Epricelist</class>
 <widget class="QWidget" name="Epricelist">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>307</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>359</width>
    <height>307</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_close">
   <property name="geometry">
    <rect>
     <x>261</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_confirm">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_TT">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>334</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string>副本掉落</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_PTJZ">
      <property name="text">
       <string>蟠桃戒指：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_PTJZ">
      <property name="maximum">
       <double>10000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="label_QTST">
      <property name="text">
       <string>齐天手套：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_QTST">
      <property name="maximum">
       <double>10000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_JLX">
      <property name="text">
       <string>金雷星：</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_KTNTH">
      <property name="text">
       <string>苦痛南天魂：</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_KTNTH">
      <property name="maximum">
       <double>10000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="2" column="2" colspan="2">
     <widget class="QLabel" name="label_HMNTH">
      <property name="text">
       <string>毁灭南天魂：</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_HMNTH">
      <property name="maximum">
       <double>10000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_SJSZ">
      <property name="text">
       <string>三极手镯：</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_SJSZ">
      <property name="maximum">
       <double>10000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_JGYD">
      <property name="text">
       <string>金刚腰带</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_JGYD">
      <property name="maximum">
       <double>10000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_MSJZ">
      <property name="text">
       <string>明神戒指</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_MSJZ">
      <property name="maximum">
       <double>10000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QLabel" name="label_DSST">
      <property name="text">
       <string>斗神手套</string>
      </property>
     </widget>
    </item>
    <item row="5" column="4">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_DSST">
      <property name="maximum">
       <double>10000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_JLX">
      <property name="maximum">
       <double>10000.000000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_FYF">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>240</y>
     <width>78</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximum">
    <double>10000.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_FYF">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>240</y>
     <width>72</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>封印符：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_clear">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>清除</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_close</sender>
   <signal>clicked()</signal>
   <receiver>Epricelist</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>537</x>
     <y>616</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
